pandas/io/common.py

```diff
@@ -397,16 +403,18 @@ def get_handle(
         # GZ Compression
         if compression == "gzip":
             if is_path:
-                f = gzip.open(path_or_buf, mode)
+                f = gzip.open(path_or_buf, mode, **compression_args)  # type: ignore
```

Comment:
Note that I added these and the below mypy ignore flags. The problem is that we're passing kwargs to functions that don't take kwargs, which is not typesafe. The alternatives I could think of both seem worse:
 - Changing the called function signatures to take an unused **kwargs parameter.
 - Explicitly extracting each parameter from compression_args and passing it individually, thereby needing to duplicate the default parameter values.